ADAL/tests/integration/AADAuthorityValidationIntegrationTests.m

```diff
@@ -54,35 +54,57 @@ - (void)tearDown
 }
 
 //Does not call the server, just passes invalid authority
-- (void)testValidateAuthorityError
+
+
+
+- (void)testValidateAuthority_whenSchemeIsHttp_shouldFailWithError
```

Comment:
I personally think result can be described in different ways: e.g shouldFailWithError is a more generic result description and shouldReturnErrorValidatedFalse describes each of the results more precisely. Both are fine from my viewpoint.